.github/workflows/release.yaml

```diff
@@ -2,7 +2,7 @@ name: Release
 on:
   push:
     tags:
-      - v*-d2iq.*
+      - v.*-d2iq.*
```

Comment:
This is not a regex, this is a glob (see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore `The branches, branches-ignore, tags, and tags-ignore keywords accept glob patterns that use characters like *, **, +, ?, ! and others to match more than one branch or tag name` and https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet) - I don't think this change is correct.